.github/workflows/mutation-tests.yml

```diff
@@ -22,7 +22,7 @@ permissions:
 jobs:
   mutations:
     name: 'mutations-${{ matrix.name }}'
-    runs-on: windows-latest
+    runs-on: ubuntu-latest
```

Comment:
I see the build is green, but I'm going to have to go spelunking through the Git history on this.

We did try using Ubuntu in the past but it didn't work correctly, so we had to go with Windows for functional correctness even though it was slower.

If whatever issue that was has been since fixed though then great.

Reply:
I found https://github.com/stryker-mutator/stryker-net/issues/2741#issuecomment-1783828972, and that points to an open issue relating to Ubuntu not working.

Maybe that's fixed now too, but still worth checking a little deeper.

Reply:
So https://github.com/App-vNext/Polly/commit/41e06e47aaf773110702349bd3c058d44e68ea5b was when we stopped using Linux because it wasn't working with .NET 8.

Either I tried it again at some point and it still didn't work, or I didn't bother because https://github.com/stryker-mutator/stryker-net/issues/2585 was still open.